css/index.css

```
@@ -282,4 +282,17 @@ footer nav a {
 .css-animation {
     animation-name: slideRight;
     animation-duration: 6s;
+}
+
+@keyframes Rotate {
```

great job on the animation
```suggestion
@keyframes Rotate {
```